src/main/kotlin/com/manimdsl/stylesheet/Stylesheet.kt

```diff
@@ -38,6 +38,7 @@ open class AnimationProperties(
 ) : StylesheetProperty()
 
 data class DefaultAnimationProperties(
+    override val borderColor: String? = "RED",
```

Comment:
Do we have to do this? I think arrays are overlapping squares and so the border color doesn't seem to work for it, can we default to null maybe?

Reply:
Ah okay sure